### ISSUE TITLE
Redirect any almost empty User:XXX pages to Participants

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython.github.io#61]
See discussion on #59.

**Comment by MarkusPiotrowski** [comment]
My suggestion from #59:
-   Many of the `User:XXX` pages (approx. 10) have no links (within biopython.org) which direct to them.
-   Many of these users are not listed on the `Participants` page, so redirecting them to it wouldn't make sense. 

So we probably should make a `User` main page (or a subheader on the `Participants` page) to direct to them.
Those without content I would simply delete (including possible links pointing to them) or redirect if there is something to direct to.

**Comment by peterjc** [comment]
Under MediaWiki the User:XXX pages were essentially user profiles for people contributing to the wiki-based website. So, if people with a User:XXX page ought to be on the participants page, lets add them to that page?

**Comment by MarkusPiotrowski** [comment]
Yes, but possibly with a simpler table/no table/empty table, because we don't have the relevant information.

**Comment by peterjc** [comment]
As @MarkusPiotrowski noted, sometimes we can just delete the user pages - e.g. https://github.com/biopython/biopython.github.io/commit/2629f7fb482461ea4041ee433601e0f7a0040d62
